### DISCUSSION TITLE
why the 🐤

### DISCUSSION THREAD

**Discussion by med-ab** [imba/imba#874] [Polls]
why the mascot is a chick

**Comment by Bohooslav** [comment]
Well, the logo is a wing, so it should be definitely a birb, maybe just a faster looking birb.

I really like the original wing

![Original ](https://raw.githubusercontent.com/imba/branding-imba/master/yellow-wing-logo/imba.svg)

**Comment by med-ab** [comment]
im okay with the wing, just not the chick
maybe if we know the meaning of 'imba' we can suggest a better icon

**Reply by haikyuu** [reply]
[Imba means for Imbalanced](https://www.urbandictionary.com/define.php?term=imba)

[Imba](https://www.urbandictionary.com/define.php?term=Imba) is used mostly in online games or games that have [patches](https://www.urbandictionary.com/define.php?term=patches) or different versions. Imba means that something is imbalanced and should be "[nerfed](https://www.urbandictionary.com/define.php?term=nerfed)" or made worse. When something is [imba](https://www.urbandictionary.com/define.php?term=imba) it means that is is too good and it makes the game unfair.
Richard: I'm not surprised you killed me Steasl [Assasin](https://www.urbandictionary.com/define.php?term=Assasin) is so imba.

[Tod](https://www.urbandictionary.com/define.php?term=Tod): Yeah they should [nerf](https://www.urbandictionary.com/define.php?term=nerf) him in the next version he's to good.

**Reply by haikyuu** [reply]
it's not a chick. It's a bird* 😆

**Reply by med-ab** [reply]
the one with the wing is niiice  
![Intersectimba](https://github.com/imba/imba/assets/77408306/657a932a-623d-4518-8e06-b401eb264ddd)
it should be consistent everywhere
![imba](https://github.com/imba/imba/assets/77408306/4e5c496d-2e2e-4ae7-9d38-df9c709fb38a)
and we can change the emoji too from:
🐤 The friendly full-stack language
to this:
🪽 The friendly full-stack language

**Reply by Bohooslav** [reply]
You don't know that in the past it used to be the way you see it.
